### PR TITLE
Copy header code from content-build template for ARF app

### PR DESCRIPTION
**Pull request by emily-gullo**
## Summary

ARP intends to operate with a distinct style as per user requirements. To this end, we are anticipating the differing styles with unique templates in `content-build`. This ticket/PR concerns the header, specifically. At this time, 

## Related issue(s)

Issue: [77781](https://app.zenhub.com/workspaces/accredited-representative-facing-team-65453a97a9cc36069a2ad1d6/issues/gh/department-of-veterans-affairs/va.gov-team/77781)

## Testing done

- Linked locally via layout template file, expecting and receiving same behavior.

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

## Acceptance criteria
- [ ]  Header code is copied into new `accredited-rep-facing` header file

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

At this point, I believe we do not need to remove code - but we will need to edit/add when we have appropriate design content that defines our specific header.  I think this will be an included part of future tickets. Appreciate any thoughts and feedback.

### CODE REVIEW

**Comment by emily-gullo** [comment]
As initially discussed in pairing, I pushed up a copy so we could assess as a team. I appreciate your feedback! I've made the requested changes for now. I have concerns with removing much more, as there is a bunch to remove or change, but not until we have our custom structure to support it (ie styles and web components) I suggest a further ticket to address these issues, or to have edits be part of styling tickets. Thoughts?

**Comment by emily-gullo** [comment]
Making additional edits, please hold up on review

**Comment by emily-gullo** [comment]
Ready for review

File: src/site/includes/accredited_representative_portal/header.html [hunk 1]
```diff
@@ -1,11 +1,123 @@
...
+  <!-- Add polyfills. -->
+  <script nonce="**CSP_NONCE**" nomodule data-entry-name="polyfills.js"></script>
+
+  <!-- Add analytics helpers. -->
+  <script nonce="**CSP_NONCE**">
+    {% include "src/site/assets/js/record-event.js" %}
+  </script>
+
+  <!-- Add web components. -->
+  {% if entityId != 11463 or buildtype != "vagovdev") %}
```
[the remark this answers is not in the document]
Comment (emily-gullo) [reply src/site/includes/accredited_representative_portal/header.html:26]: I was wondering what that entity was! TY! I'm on it.

File: src/site/includes/accredited_representative_portal/header.html [hunk 2]
```diff
@@ -1,11 +1,123 @@
...
+    <link rel="stylesheet" data-entry-name="web-components.css">
+    {% comment %}
+      The `useLocalStylesAndComponents` registry value allows apps to opt-out of
+      this global file which will define all web components. Instead, they will manually
+      define a subset of web components which are used in their application
+    {% endcomment %}
+    {% unless useLocalStylesAndComponents %}
+      <script nonce="**CSP_NONCE**" defer data-entry-name="web-components.js"></script>
+    {% endunless %}
```
[the remark this answers is not in the document]
Comment (emily-gullo) [reply src/site/includes/accredited_representative_portal/header.html:35]: Wasn't sure if we were an exception here due to the different styles, but that makes a lot of sense, thanks for clarifying! :)

File: src/site/includes/accredited_representative_portal/header.html [hunk 3]
```diff
@@ -1,11 +1,123 @@
...
+  <link rel="preload" href="/generated/fa-solid-900.woff2" as="font" type="font/woff2" crossorigin>
+
+  <!-- CSS -->
+  {% comment %}
+    Apps that opt out of the site-wide stylesheet must include a smaller subset of styles
+    in their application bundle
+  {% endcomment %}
+  {% unless useLocalStylesAndComponents %}
+    <link rel="stylesheet" data-entry-name="style.css">
+  {% endunless %}
```
[the remark this answers is not in the document]
Comment (emily-gullo) [reply src/site/includes/accredited_representative_portal/header.html:58]: 👍

**Review by randimays — APPROVED** [review]
LGTM

**Pull request merged**